scripts/style.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+find . \
+    -name 'third_party' -prune -o \
+    -name 'Auth' -prune -o \
+    -name 'AuthSamples' -prune -o \
+    -name 'Database' -prune -o \
+    -name 'FirebaseCommunity.h' -prune -o \
+    -name 'Messaging' -prune -o \
+    -name 'Storage' -prune -o \
+    -name '*.[mh]' \
+    -not -name '*.pbobjc.*' \
+    -exec clang-format -style=file -i '{}' \;
```

Comment:
This will be much, much faster if you change this from
```
 -exec foo \;
```
To 
```
  -print0 | xargs -0 foo
```
The former forks foo once per file. The latter forks foo only as many times as dividing all files in ARG_MAX requires (though it requires foo to take many files as an argument, which clang-format does).